launch_crab_multithreading.py

```diff
@@ -835,10 +1087,28 @@ def submit(config, dryrun = False):
         string_GT = 'PGT='+str(GT)
         string_JECstring = 'PJECstring='+str(JECstring)
         string_JERstring = 'PJERstring='+str(JERstring)
+        string_MuonSFIDstring = 'PMuonSFIDstring='+str(MuonSFIDstring)
+        string_MuonSFISOstring = 'PMuonSFISOstring='+str(MuonSFISOstring)
+        string_MuonSFTriggerstring = 'PMuonSFTriggerstring='+str(MuonSFTriggerstring)
+        string_eleVetoIDstring = 'PeleVetoIDstring='+str(eleVetoIDstring)
+        string_eleLooseIdstring = 'PeleLooseIdstring='+str(eleLooseIdstring)
+        string_eleMediumIdstring = 'PeleMediumIdstring='+str(eleMediumIdstring)
+        string_eleTightIdstring = 'PeleTightIdstring='+str(eleTightIdstring)
+        string_eleMVA90noISOstring = 'PeleMVA90noISOstring='+str(eleMVA90noISOstring)
+        string_eleMVA80noISOstring = 'PeleMVA80noISOstring='+str(eleMVA80noISOstring)
+        string_phoLooseIdFilestring = 'PphoLooseIdFilestring='+str(phoLooseIdFilestring)
+        string_phoMediumIdFilestring = 'PphoMediumIdFilestring='+str(phoMediumIdFilestring)
+        string_phoTightIdFilestring = 'PphoTightIdFilestring='+str(phoTightIdFilestring)
+        string_phoMVANonTrigMediumIdFilestring = 'PphoMVANonTrigMediumIdFilestring='+str(phoMVANonTrigMediumIdFilestring)
+        string_btagSFstring = 'PbtagSFstring='+str(btagSFstring)
```

Comment:
Unless the same SF files can/will be used also in the AODNtuplizer, these filenames could be set directly in the config file. This is already implemented for `RunLocal=True` anyways, and having them in one place only would make updating/bug fixing easier. Maybe @lbenato can comment?

Reply:
But then one need a per-era switch in the config?

Reply:
There's already one, with the exact same lines:
https://github.com/lbenato/LLP2018/blob/test_tracking17Nov2020/python/Ntuplizer2018.py#L663-L730

One would just need to move it outside the `RunLocal`

Reply:
Okay good then!

Reply:
Actually, this might be worth keeping in the crab launch script together with the reminder to fix the 2018 muon SF. We can remove the additional parameters afterwards, together with the warning.